packages/storage/src/implementation/metadata.ts

```diff
@@ -37,27 +39,27 @@ export function noXform_(metadata: Metadata, value: any): any {
 export class Mapping {
   local: string;
   writable: boolean;
-  xform: (p1: Metadata, p2: any) => any;
+  xform: (p1: Metadata, p2: unknown) => unknown;
 
   constructor(
     public server: string,
-    opt_local?: string | null,
-    opt_writable?: boolean,
-    opt_xform?: (p1: Metadata, p2: any) => any | null
+    local?: string | null,
+    writable?: boolean,
+    xform?: ((p1: Metadata, p2: unknown) => unknown) | null
   ) {
-    this.local = opt_local || server;
-    this.writable = !!opt_writable;
-    this.xform = opt_xform || noXform_;
+    this.local = local || server;
+    this.writable = !!writable;
+    this.xform = xform || noXform_;
   }
 }
 type Mappings = Mapping[];
 
 export { Mappings };
 
 let mappings_: Mappings | null = null;
-
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function xformPath(fullPath: any): string {
-  let valid = type.isString(fullPath);
+  const valid = type.isString(fullPath);
   if (!valid || fullPath.length < 2) {
     return fullPath;
```

Comment:
Based on the function, it could return non-string value. Not sure what the right fix is. @schmidt-sebastian, What do you think?

Reply:
I am not 100% certain, but I tried to fix the types in this file. I got it to work without `any` in the follow up. Wasn't pleasant :)

Reply:
Awesome! I was so frustrated by this file that I initially disabled eslint for it entirely